auth0/resource_data.go

```diff
@@ -92,6 +92,17 @@ func Slice(d Data, key string) (s []interface{}) {
 	return
 }
 
+// Set accesses the value held by key and type asserts it to a set.
+func Set(d Data, key string) (s *schema.Set) {
+	if d.HasChange(key) {
+		v, ok := d.GetOkExists(key)
+		if ok {
+			s = v.(*schema.Set)
+		}
+	}
+	return
```

Comment:
`Set().List()` will throw a nil pointer exception if the field hasn't changed.

Reply:
Ouch, good catch, fixed this in 6a6544c.